src/v1/profile/info/ProfileInfoPresenter.tsx

```diff
@@ -0,0 +1,43 @@
+import type { APIUser } from '@/types/user';
+import Avatar from '@/v1/base/Avatar';
+import Badge from '@/v1/base/Badge';
+
+type ProfileInfoProps = Pick<APIUser, 'nickname' | 'profileImage' | 'job'>;
+
+// COMMENT: 프로필 정보 조회 API 스키마 변경으로 email, oauthnickname props 제거
+const ProfileInfoPresenter = ({
+  nickname,
+  profileImage,
+  job: { jobGroupKoreanName, jobNameKoreanName },
+}: ProfileInfoProps) => {
+  return (
+    <div className="flex flex-col gap-[2rem]">
```

Comment:
p2;

프로필 정보와 책장/모임 UI가 구분되면 좋을 것 같아서 디자인할 때 사이에 2rem의 여백을 더 줬었어요! 코드에 반영해주시면 감사하겠습니다.. 🙏🏻

```suggestion
    <div className="flex flex-col gap-[2rem] mb-[2rem]">
```

<img width="456" alt="스크린샷 2023-12-12 오후 9 11 21" src="https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/57716832/f8a2a254-9e23-484c-8341-f1bc40b62903">